### DISCUSSION TITLE
How should I add quorum to my proposal

### DISCUSSION THREAD

**Discussion by metrox-eth** [snapshot-labs/snapshot#1175] [Q&A] [answered]
Suggest answers below or vote for an answer.

**Comment by metrox-eth** [comment, marked as answer]
### Quorum setting

You can enter a value in the quorum field 

<img width="628" alt="Capture d’écran 2021-12-05 à 14 44 43" src="https://user-images.githubusercontent.com/55286013/144751671-f8fdea69-879c-4260-a393-b4962c43e2b1.png">

Then on the UI, based on the amount of voting power used to vote, you'll see the % that you've reached of the quorum.
For example
https://snapshot.org/#/yam.eth/proposal/0x21ea31e896ec5b5a49a3653e51e787ee834aaf953263144ab936ed756f36609f 
<img width="1110" alt="Capture d’écran 2021-12-05 à 15 56 35" src="https://user-images.githubusercontent.com/55286013/144751876-470a3a01-cdfe-41eb-bc6f-924e55f98fb8.png">

### Quorum plugin
With the quorum  plugin you define a quorum, it can be a fixed number or a kind of custom-call function to get a number dynamicly onchain.

You can setup it with a static number using params in this format:
```
{
  "strategy": "static",
  "total": 200000
}
```

Example with Yam here: 
[https://hub.snapshot.org/api/spaces/yam.eth](https://hub.snapshot.org/api/spaces/yam.eth)

<img width="297" alt="Capture d’écran 2021-12-05 à 15 46 43" src="https://user-images.githubusercontent.com/55286013/144751740-fec17ce9-4745-49cf-a927-9d4e0fca7bd4.png">

Or you can use contract call to get a dynamic value with this format:
[https://hub.snapshot.org/api/spaces/bancornetwork.eth](https://hub.snapshot.org/api/spaces/bancornetwork.eth)

<img width="495" alt="Capture d’écran 2021-12-05 à 15 47 23" src="https://user-images.githubusercontent.com/55286013/144751744-50a26c2f-6284-404b-bd15-3a5d3dc06952.png">

**Reply by samwhosung** [reply]
Looks good! One thing that could be more clear is that the UI also shows quorum differently depending on if it is a plugin or just a setting.
